ceph/ceph_admin/apply.py

```diff
@@ -71,63 +71,43 @@ def apply(self: ServiceProtocol, config: Dict) -> None:
 
         args = config.get("args")
 
-        node_names = None
         verify_service = False
         placement = args.pop("placement", {})
 
         if placement:
             placement_str = "--placement="
+            sep = placement.get("sep", " ")
+            node_names = list()
             verify_service = True
 
             if "label" in placement:
-                label = placement["label"]
-                node_names = [
-                    node["hostname"] for node in self.get_hosts_by_label(label)
-                ]
-                placement_str += f'"label:{label}"'
-                base_cmd.append(placement_str)
+                node_names.append(f"label:{placement['label']}")
 
             if "nodes" in placement:
                 nodes = placement.get("nodes")
 
                 if "*" in nodes:
-                    placement_str += '"*"'
-                    node_names = list()
-                    for node in self.cluster.node_list:
-                        if (
-                            len(node.role.role_list) == 1
-                            and ["client"] == node.role.role_list
-                        ):
-                            continue
-                        node_names.append(node.shortname)
+                    node_names.append("*")
```

Comment:
Shouldn't we exclude the `client`

Reply:
Actually its not required, as `"*"` should consider the nodes which is already part of the cluster. 
Earlier we used to find the nodes and replacing the nodes against `"*"` which was used same for validation also. but currently validation is different, so it would not be required.

Reply:
Okay